wire-compiler/src/test/java/com/squareup/wire/schema/LinkerTest.kt

```diff
@@ -171,6 +172,72 @@ class LinkerTest {
     assertThat(schema.protoFile("b.proto")!!.javaPackage()).isEqualTo("com.squareup.b")
   }
 
+  @Test
+  fun linkerThrowsSchemaExceptionIfProto3EnumIsMissingTagZero() {
```

Comment:
Let's add a happy proto3 path and another one which fails if 0 isn't the first element.